src/components/displaySettings/displaySettings.template.html

```diff
@@ -131,6 +131,18 @@ <h2 class="sectionTitle">
         <select id="selectTheme" is="emby-select" label="${LabelTheme}"></select>
     </div>
 
+    <div class="checkboxContainer checkboxContainer-withDescription followSystemTheme hide">
```

Comment:
What do you think if replace the checkbox with just a simple dropdown?
![image](https://user-images.githubusercontent.com/17064712/88431937-947e1e00-cdfb-11ea-8929-e1382d6d6783.png)

Reply:
If there was only two themes, and three options (device, light, dark) then a dropdown would be better, but because there is a total of 5 themes, I think that it is better to have a checkbox to enable following of system theme, and then a dropdown to choose the prefered dark theme

Reply:
Okei that makes sense :+1: